Makefile

```diff
@@ -47,6 +58,7 @@ gen_grpc:						## Generates GRPC Dependencies
 
 update_python: create_venv gen_grpc 			## Updates the python package locally
 	cd test_venv/bin && source activate
+	cd ../../
```

Comment:
if we are change back directories then I would recommend `source test_venv/bin/activate` on the previous line

Reply:
Actually gonna remove this. its not the proper way to access the virtual environment with the makefile anyways